### ISSUE TITLE
LESS compatibility

### ISSUE THREAD

**Issue by ghost** [i-akhmadullin/Sublime-CSS3#44]
Is it compatibile with LESS preprocessor ? If so, how can I achieve the mix of CSS3 syntax + LESS ?

Below screenshots after installation of `Sublime-CSS3` package.

LESS Syntax:
![image](https://cloud.githubusercontent.com/assets/19711901/21544961/e63ef948-cdd3-11e6-90d2-f8d9c31d668f.png)

CSS3 Syntax (same file):
![image](https://cloud.githubusercontent.com/assets/19711901/21544968/f7ac6ff8-cdd3-11e6-844c-a0e1145f5e0a.png)

**Comment by mahmoudzadah** [comment]
Yes, same story with SCSS. I think the only option is to choose one at a time.
